scripts/enforce-indent-two-spaces-outside-java.awk

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env awk
```

Comment:
I do not understand this language XD looks fine

Reply:
nice to have it extracted!

Reply:
`awk` is love, `awk` is life